taskclient/__init__.py,sha256=dY23pkGcRpLDfNwr_lgxmfYnSMTyWialr4c3NgKowMc,1301
taskclient/connection.py,sha256=pTrLTbvzzaPjVLAMgBtKg9jYWaL9zVg9orBt9A2so0Q,17615
taskclient/drivers.py,sha256=dpMu7IFYcvYEHdAx3xsnEFYq_l60OlnLwyNMpPcbU9A,944
taskclient/exceptions.py,sha256=hq7LmKV3Uq5FdcSg3Vh7UysCqvJffxG7mSorY-6Wnus,904
task_client-0.2.0.dist-info/LICENSE,sha256=ohsbe-1180tqV0K-E81i4S5MMEHucR1lLirw3LvIpXw,665
task_client-0.2.0.dist-info/METADATA,sha256=JENrguKUvVzqkO26oFL9alCk6FwwGvz7N8GuUAjU6r0,1352
task_client-0.2.0.dist-info/WHEEL,sha256=CbUdLTqD3-4zWemf83rgR_2_MC4TeXw9qXwrXte5w4w,92
task_client-0.2.0.dist-info/entry_points.txt,sha256=eqGnTHtEVMYwfXVk1Z9MhC8O2N8wuqAbr0lisLmrkxs,20
task_client-0.2.0.dist-info/top_level.txt,sha256=iW8O438sdBiWDCHlB0DFKW5ZOtxdKmvcA2_NuCDuUaA,11
task_client-0.2.0.dist-info/RECORD,,
